SDPi_Supplement/asciidoc/volume2/dev-25/tf2-dev-25.adoc

```diff
@@ -0,0 +1,68 @@
+[#vol2_clause_dev_25]
+
+=== Discover BICEPS Services [DEV-25]
+
+==== Scope
+[sdpi_transaction#transaction-dev-25,actors='actor-somds-provider actor-somds-consumer',figure=vol2-figure-dev-25-sequence,messages='message-discover-biceps-services']
+
+This transaction is used to exchange the resources metadata between a <<actor_somds_provider>> and a <<actor_somds_consumer>>.
+
+==== Actor Roles
+
+.Actor Roles DEV-25
+[cols="1,2"]
+|===
+|Actor |Roles
+
+|<<actor_somds_consumer>>
+| Asks for the resource representation of the <<actor_somds_provider>> via the <<vol2_clause_dev_25_message_get_metadata>> messages.
+
+|<<actor_somds_provider>>
+| Sends its resource representation to the <<actor_somds_consumer>> via the <<vol2_clause_dev_25_message_get_metadata_response>> messages.
+
+|===
+
+==== Referenced Standards
+
+* <<ieee-11073-10207-2017>> Section 9 Discovery Model
+
+==== Messages
+.Message Interaction Diagram DEV-25
+[plantuml#vol2_figure_dev_25_sequence, target=puml-dev-25-sequence, format=svg, reftext='{figure-caption} {counter:refnum}',width=100%]
+
+....
+include::../../plantuml/vol2-figure-dev-25-sequence.puml[]
+....
+
+[#vol2_clause_dev_25_message_get_metadata]
+===== GetMetadata Message
+The GetMetadata message is used by a <<actor_somds_consumer>> to request the resource representation data of a <<actor_somds_provider>>.
+
+====== Trigger Events
+This message is sent when a <<actor_somds_consumer>> has discovered a <<property_transport_address>> for a <<actor_somds_provider>>.

```

Comment:
Please double check existence of `property_transport_address`. It does not appear in the generated HTML output. Take a look at `asciidoc/volume0/tf0-ch-d-glossary.adoc` and append the term if missing. Also please check other transactions for the occurrence of `physical address`, which if I recall correctly is the term I used - it should be linked accordingly then (DEV-23 and DEV-24).